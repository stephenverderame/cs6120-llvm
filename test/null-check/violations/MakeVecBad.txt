[31mSafety Violation[00m: Use of potentially null pointer in load at MakeVecBad.cc:20
