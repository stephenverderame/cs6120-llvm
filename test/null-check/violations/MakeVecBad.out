[31mSafety Violation[00m: Use of potentially null pointer in store at MakeVecBad.cc:17
[31mSafety Violation[00m: Use of potentially null pointer in load at MakeVecBad.cc:20
